Tools/Scripts/libraries/webkitscmpy/webkitscmpy/test/git_unittest.py

```diff
@@ -614,6 +614,57 @@ def test_is_suitable_branch_for_pull_request(self):
             self.assertEqual(repo.is_suitable_branch_for_pull_request('branch-a', source_remote=source_remote), False)
             self.assertEqual(repo.is_suitable_branch_for_pull_request(None, source_remote=source_remote), False)
 
+    def test_last_commits_on(self):
```

Comment:
This test should be most similar to `test_files_changed`. You'll just need the `with mocks.local.Git(self.path), OutputCapture():` as your mock, `local.last_commits_on` won't "just work", though, you're going to need to mock the `git` command you're calling in `Tools/Scripts/libraries/webkitscmpy/webkitscmpy/mocks/local/git.py`

Reply:
Okay I updated following your directions.